jupyter_server/services/contents/manager.py

```diff
@@ -121,6 +133,8 @@ def _notary_default(self):
     """,
     )
 
+    max_copy_folder_size_mb = Int(500, config=True, help="The max folder size that can be copied")
```

Comment:
Hmm, perhaps this trait should be on `FileManager`, since it isn't implemented in the more generic `ContentsManager`.  Anyone have any thoughts?

Reply:
Yes - I completely agree - good catch.

Reply:
I will make sure to make that change @blink1073.

Reply:
@blink1073, when you say `FileManager` do you mean the `FileContentsManager` or the `LargeFileManager`?

Reply:
Hi @kenyaachon - yes, `FileContentsManager` is the appropriate location for the trait since it's the "most base" class for the file-based contents managers.  Sorry for the confusion.